Eastern-Research-Group/csb-rebate-forms-app
# Feature/implement deduplication web service

## Related Issues:
* CSBAPP-332

## Main Changes:
Initial implementation of the wrapper app to support BAP de-duplication efforts. From the wrapper side, it's really just a proxy for the BAP record matcher web service. This is a quick initial MVP to allow Formio form development to begin, but we'll want to apply some additional checks in the wrapper app's web service before making the request against the BAP.

## Steps To Test:
1. Post some 'duplicate' contact or org data to the web service endpoint (be sure to include the x-api-key header) and ensure you're getting a response
2. Ensure you get an appropriate error message if you don't include the proper api key header

## TODO:
- [ ] Update web service to perform checks (TBD) before issuing request against the BAP (e.g., checking shape of post body, ensuring certain fields exist, ensuring post body isn't too large, etc.)
